charts/internal/shoot-system-components/charts/csi-driver-node/values.yaml

```diff
@@ -36,4 +36,4 @@ resources:
     limits:
       memory: 300Mi
 
-pspDisabled: false
+pspDisabled: false
```

Comment:
Nit: In my previous review I commented about the empty trailing lines because there were more than 1. I think it is accepted to leave 1 empty line at the end of the file. You could revert the no-op change to this file.